### PR TITLE
remove invalid chars

### LINKED ISSUE #3068: # Unrecognized character "�";
# Issue Title

Be as descriptive as you can with your title. If your issue requests
a new topic, preface the title with [NewTopic].

# General

We use issues to drive the discussion for changes to
existing topics and the creation of new topics.

This way, we have those discussions before significant
effort is spent writing a new topic. Click on the Guidelines
for Contributing link above for details.

## Issues with Existing Topics

If the issue is a simple typo or similar correction, you can just submit a PR.
Otherwise, let us know what's wrong or what we should improve. Include a
link to the topic.

## Requests for new Topics

1. Tell us where this topic should go in the Table of Contents.

   - Consider where you looked for this information before opening an issue.

2. Write an abstract

   - In one **short** paragraph, describe what this topic will cover.

3. Fill in an outline

    - Write a complete outline for the new topic. We'll help review the outline and approve that before anyone writes a topic.

4. Suggest reviewers

    - If you know someone who can provide feedback, use '@' to ask them to review.

### PR DESCRIPTION
**Pull request by BillWagner**
Fixes #3068 
These characters cause issues with localization

### CODE REVIEW

**Review by mairaw — APPROVED** [review]
LGTM

File: samples/snippets/visualbasic/VS_Snippets_Misc/tpl_synccontext/vb/MainWindow.xaml.vb [hunk 1]
```diff
@@ -48,7 +48,7 @@ Partial Public Class MainWindow : Inherits Window
             images(x) = Task.Factory.StartNew(Function() LoadImage(files(x)))
         Next
 
-        ' When they�ve all been loaded, tile them into a single byte array.
+        ' When they have all been loaded, tile them into a single byte array.
```
Comment (svick) [inline samples/snippets/visualbasic/VS_Snippets_Misc/tpl_synccontext/vb/MainWindow.xaml.vb]: Tiny nitpick: why does C# use "they've", but VB uses "they have"?

Comment (BillWagner) [reply samples/snippets/visualbasic/VS_Snippets_Misc/tpl_synccontext/vb/MainWindow.xaml.vb]: I just try and avoid the ' character in VB comments. I know it's not strictly necessary, but it just looks bad to my eyes.  🤷‍♂️

**Review by svick — APPROVED** [review]

**Comment by mairaw** [comment]
This build wasn't showing up in the portal.

**Pull request merged**